.github/CODEOWNERS

```diff
@@ -1,51 +1,3 @@
-# Last match takes precedence.
-
-# Default code owners when none specified
-*                                @warner @Chris-Hibbert @FUDCo @michaelfig
-
-# Tooling and dependencies
-/*                               @kriskowal @michaelfig
-/.github/                        @kriskowal @michaelfig
-/docker/                         @kriskowal @michaelfig
-/patches/                        @kriskowal @michaelfig
-/scripts/                        @kriskowal @michaelfig
-/yarn.lock                       @mhofman
-
-# Golang parts of the repo
-/go.*                            @JimLarson
-/golang/                         @JimLarson
-
-# Each npm package
-/packages/ERTP/                  @erights @Chris-Hibbert
-/packages/SwingSet/              @warner @FUDCo
-/packages/access-token/          @kriskowal @michaelfig
-/packages/agoric-cli/            @kriskowal @michaelfig
-/packages/assert/                @kriskowal @turadg @michaelfig @erights
-/packages/cosmic-swingset/       @michaelfig
-/packages/deploy-script-support/ @michaelfig
-/packages/deployment/            @michaelfig
-/packages/eslint-config/         @kriskowal @michaelfig @erights @turadg @mhofman
-/packages/governance/            @Chris-Hibbert
-/packages/import-manager/        @michaelfig
-/packages/notifier/              @kriskowal @turadg @michaelfig @erights
-/packages/pegasus/               @michaelfig
-/packages/run-protocol           @Chris-Hibbert @turadg
-/packages/same-structure/        @kriskowal @turadg @michaelfig @erights
-/packages/sharing-service/       @michaelfig @Chris-Hibbert
-/packages/solo/                  @kriskowal @michaelfig
-/packages/sparse-ints/           @kriskowal @turadg @michaelfig
-/packages/spawner/               @michaelfig
-/packages/stat-logger/           @warner @FUDCo
-/packages/store/                 @warner @FUDCo @erights
-/packages/swing-store/           @warner @FUDCo
-/packages/swingset-runner/       @FUDCo
-/packages/tame-metering/         @kriskowal @michaelfig
-/packages/telemetry              @kriskowal @michaelfig
-/packages/ui-components/         @michaelfig @samsiegart  @turadg
-/packages/vats/                  @michaelfig
-/packages/vat-data/              @turadg
-/packages/wallet/                @michaelfig @samsiegart
-/packages/wallet-connection/     @michaelfig @samsiegart
-/packages/web-components/        @samsiegart
-/packages/xsnap/                 @warner @dckc @kriskowal
-/packages/zoe                    @erights @Chris-Hibbert
+# We are currently only using CODEOWNERS for a small subset of our packages.
```

Comment:
I think we want people to feel free to add themselves. Don't know if that merits a comment.